src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SyntaxFacts/CSharpSyntaxFacts.cs

```diff
@@ -1676,6 +1676,14 @@ public void GetPartsOfObjectCreationExpression(SyntaxNode node, out SyntaxNode t
             initializer = objectCreationExpression.Initializer;
         }
 
+        public void GetPartsOfImplicitObjectCreationExpression(SyntaxNode node, out SyntaxToken keyword, out SyntaxNode argumentList, out SyntaxNode? initializer)
+        {
+            var implicitObjectCreationExpression = (ImplicitObjectCreationExpressionSyntax)node;
+            keyword = implicitObjectCreationExpression.NewKeyword;
```

Comment:
nit.  this is only a c# case, and we know in c# it's always the first token.  so could literally just hardcode taht at the use site.  But i don't feel strongly.  if you want to do it this way, that's fine with me as well :)

Reply:
I think the use site doesn't have access to C# syntax, so I'll keep this one. To me it worked since it matches the existing `GetPartsOfObjectCreationExpression` method.